### PR TITLE
Hello Microverse project pull request

### PR DESCRIPTION
**Pull request by RichardKaranuMbuti**
This is a simple html and css project to access my knowledge on git

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi  @RichardKaranuMbuti,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights


## Required Changes ♻️
_Check the comments under the review._
- [x] setup your readme file in accordance to [Microverse template](https://github.com/microverseinc/readme-template)
- [x] Kindly always make a descriptive PR which would have a heading and the things you did under it. This is for people to know what you have done a a glance.
- [x] It is required you set up a test.md file and add it to gitignore file.

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,192 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (huseyinbiyik) [inline README.md:190]: As previous reviewer requested, please use the template and replace the placeholders with your project data. Here, MIT license link is not working.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @RichardKaranuMbuti,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linters check passes 🟢


 ## Required Changes ♻️
_Check the comments under the review._
- You have a conflict needed to be solved on your branch. Please solve them before submitting a new request.
- As the previous reviewer requested before please always make a descriptive PR which would have a heading and the things you did under it. This is for people to know what you have done a glance. You check to learn more about in this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Write%20a%20meaningful%20title%20for%20your%20pull%20request%20and%20always%20add%20a%20short%20summary%20of%20changes%20introduced%20in%20the%20pull%20request.).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,271 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```
Comment (Tufahel) [inline README.md:97]: ***[OPTIONAL]*** `Live Demo Link` means the live version of your website that can be accessed from anywhere in the world. 
Kindly consider adding the correct link, as it is professional, and the viewer can easily find your website's online version from Readme. Here is a [FreecodeCamp Link](https://www.freecodecamp.org/news/publish-your-website-netlify-github/) for reference on how to do it.

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @RichardKaranuMbuti 

Great work on implementing the previous reviewer's suggestions.
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: No linter errors.
:heavy_check_mark: The PR summary and commit messages are very descriptive.
:heavy_check_mark: Correct Readme format.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**